webpack/assets/javascripts/react_app/common/__snapshots__/urlHelpers.test.js.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`URI query and stringify tests should return stringified params 1`] = `"?page=3&per_page=21&search=search"`;
+
+exports[`URI query and stringify tests should test params functions 1`] = `
+Object {
+  "page": 1,
+  "perPage": NaN,
+  "searchQuery": "",
```

Comment:
Can we simulate real values for the tests?
Maybe by updating the `window.location`?